crates/templates/src/writer.rs

```diff
@@ -9,6 +9,7 @@ pub(crate) struct TemplateOutputs {
 pub(crate) enum TemplateOutput {
     WriteFile(PathBuf, Vec<u8>),
     AppendToml(PathBuf, String),
+    MergeToml(PathBuf, &'static str, String), // only have to worry about merging to top level for now
```

Comment:
top-level = merging into the root TOML table rather than having to merge into the `component` table or the `component.trigger` table.  Is there a better way to express this?  I couldn't think of one

Reply:
I'm trying "root table" instead because at least that hints to the reader that it's a TOML term of art

Reply:
Maybe "app-level"?